README.md

```diff
@@ -27,14 +27,18 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
+   the DOM is contains a webpage when it is built, and has different nodes that have built in methods that manipulate the html.
 
 2. What is an event?
+   A response when a user clicks somewhere .
 
 3. What is an event listener?
+   takes to arguments, the listens for the event. The second is a callback to fire when the event is triggered.
 
 4. Why would we convert a NodeList into an Array?
-
+   to use forEach to iterate over data.
```

Comment:
we can actually use .forEach() with a NodeList, but we need an Array to use methods such as .map()